c2corg_api/models/user.py

```diff
@@ -66,6 +67,8 @@ class User(Base):
     validation_nonce_expire = Column(DateTime, nullable=True, unique=False)
     _password = Column('password', String(255), nullable=False)
 
+    lang = Column(String(2), nullable=False, default='fr')
```

Comment:
This should be a foreign-key:
https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/document.py#L223